frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -119,18 +144,31 @@ function BookInfo:show(file, book_props)
     end
 
     local KeyValuePage = require("ui/widget/keyvaluepage")
-    local widget = KeyValuePage:new{
+    self.kvp_widget = KeyValuePage:new{
         title = _("Book information"),
         value_overflow_align = "right",
         kv_pairs = kv_pairs,
         values_lang = values_lang,
+        close_callback = function()
+            if self.updated then
+                if caller_callback then
+                    caller_callback()
+                end
+                local FileManager = require("apps/filemanager/filemanager")
+                if FileManager.instance then
+                    FileManager.instance:onRefresh()
+                end
```

Comment:
(Architecturally, shouldn't the FileManager stuff be part of the job done in the `caller_callback` ? ie. in classic mode, you wouldn't need to refresh filemanager, right?)

Reply:
I have sdr folder visible (patched filechooser), so adding/removing the custom cover for not-yet-opened books requires refreshing.

Reply:
So may be add a comment `-- not strictly needed if not using coverbrowser, but may help while developping`.
Just so that we don't wonder why it's there in the future.